python/cuml/cuml/tsa/batched_lbfgs.py

```diff
@@ -175,12 +175,11 @@ def fprime_f(x):
                     wa[ib],
                     iwa[ib],
                     task[ib],
-                    iprint,
```

Comment:
Should we also remove this from the functions' signature if it has no more effect?